src/js/components/uploadFabsFile/UploadFabsFileValidation.jsx

```diff
@@ -364,36 +406,40 @@ export class UploadFabsFileValidation extends React.Component {
                                         <Icons.InfoCircle />
                                         <span className="tooltip-popover above">
                                             <span>
-                                                The published file differs from the submitted file in four ways:
+                                                    The published file differs from the submitted file in four ways:
                                             </span>
-                                            <span>1) It contains derivations based on agency data, as described
-                                        in the DAIMS Practices and Procedures document;
+                                            <span>
+                                                    1) It contains derivations based on agency data, as described in the
+                                                    DAIMS Practices and Procedures document;
                                             </span>
 
-                                            <span>2) Any rows in the submitted file with unresolved critical
-                                        errors will not be published.
+                                            <span>
+                                                    2) Any rows in the submitted file with unresolved critical errors
+                                                    will not be published.
                                             </span>
 
-                                            <span>3) Its order matches the header order in DAIMS-IDD-D2,
-                                        rather than that in the submitted file.
+                                            <span>
+                                                    3) Its order matches the header order in DAIMS-IDD-D2, rather than
+                                                    that in the submitted file.
                                             </span>
 
-                                            <span>4) Any extraneous headers, Flex or otherwise, are not carried
-                                         over to the published file.
+                                            <span>
+                                                    4) Any extraneous headers, Flex or otherwise, are not carried over
+                                                    to the published file.
                                             </span>
                                         </span>
                                     </span>
                                 </div>
-                                <button
-                                    className="pull-right col-xs-3 us-da-disabled-button"
-                                    disabled> Download Published File
+                                <button className="pull-right col-xs-3 us-da-disabled-button" disabled>
+                                    {' '}
```

Comment:
What exactly is this doing? (curious)

Reply:
Just adding white space, which seems unnecessary. Removing!